docs/devel/api_changes.md

```diff
@@ -292,6 +292,33 @@ consumers, please send an announcement to `kubernetes-dev@googlegroups.com` befo
 the change gets in. If you are unsure, ask. Also make sure that the change gets documented in
 `CHANGELOG.md` for the next release.
 
+## Defining new types
+
+Prefer to group together related concepts into a sub-struct, rather than having lots of fields at
```

Comment:
Of course it is appropriate when it is appropriate.  But that is very much a style thing.  Like, this example, I would argue does NOT benefit from it.